app/forms/base_form.rb

```diff
@@ -60,6 +60,10 @@ def new_record?
     true
   end
 
+  def split_address?
+    c100_application.version >= 3 || ENV.key?('SPLIT_ADDRESS')
```

Comment:
Let's add another `or` for ENV.key?('DEV_TOOLS_ENABLED') so once deployed to staging the feature is enabled automatically and the team can start testing it.